### ISSUE TITLE
param lose

### ISSUE THREAD

**Issue by eliganel** [AFNetworking/AFNetworking#4001]
I used the 2.+ for the two request who needs keep sessions,and it worked.But when I changed it to 3.1.0,the first request is allright,the second failed,and I saw the request on the server had no parameters,all the parameters became null,what happened? Help!Help!Help!

**Comment by qclmnt** [comment]
I just experienced the same issue.

Version of AFNetworking: 3.1.0

The method that I use : 
```
- (NSURLSessionDataTask *)PUT:(NSString *)URLString
                   parameters:(id)parameters
                      success:(void (^)(NSURLSessionDataTask *task, id responseObject))success
                      failure:(void (^)(NSURLSessionDataTask *task, NSError *error))failure
```

when a I have an empty dictionary into my parameters it desappear into :
```
- (NSURLSessionDataTask *)dataTaskWithHTTPMethod:(NSString *)method
                                       URLString:(NSString *)URLString
                                      parameters:(id)parameters
                                  uploadProgress:(nullable void (^)(NSProgress *uploadProgress)) uploadProgress
                                downloadProgress:(nullable void (^)(NSProgress *downloadProgress)) downloadProgress
                                         success:(void (^)(NSURLSessionDataTask *, id))success
                                         failure:(void (^)(NSURLSessionDataTask *, NSError *))failure
```
when we do : `NSMutableURLRequest *request = [self.requestSerializer requestWithMethod:method URLString:[[NSURL URLWithString:URLString relativeToURL:self.baseURL] absoluteString] parameters:parameters error:&serializationError];`

It seams to be a serialization problem.

Exemple of params :
```
{
apiKey: 32334635Gfzgegergerg235345gegd
deepness: 0
ids: {}
}
```
results body : ` apikey=32334635Gfzgegergerg235345gegd&deepness=0 ` without ids...

**Comment by qclmnt** [comment]
Here what i did : https://github.com/Youmiam/AFNetworking/commit/f1a4cb194eb4ab9a77bd1707f39a0806ba324cba

I just made this fix from the 3.1.0 release.

**Comment by SlaunchaMan** [comment]
For general help, please ask [on StackOverflow](https://stackoverflow.com/questions/tagged/afnetworking).
